client_cli/Cargo.toml

```diff
@@ -21,11 +21,12 @@ iroha_client = { version = "=2.0.0-pre-rc.11", path = "../client" }
 iroha_data_model = { version = "=2.0.0-pre-rc.11", path = "../data_model" }
 iroha_crypto = { version = "=2.0.0-pre-rc.11", path = "../crypto" }
 iroha_config = { version = "=2.0.0-pre-rc.11", path = "../config" }
+iroha = { version = "=2.0.0-pre-rc.11", path = "../cli" }
 
 color-eyre = "0.6.2"
 clap = { version = "3.2.16", features = ["derive"] }
 dialoguer = { version = "0.10.2", default-features = false }
-serde_json = "1.0.83"
+json5 = "0.4.1"
```

Comment:
Is there any other crate where we technically don't need full `serde_json` functionality and could make a switch like this?

Reply:
Good idea, but I can't find any more. It's related only to *cli* things like `cli` and `client_cli` crates. It's worked for `client_cli` here, but doesn't work for `cli`, because `torrii` inside it uses `serde_json`.

Also `json5` it-self depends on `serde_json` so we won't gain much of it